openedx/core/lib/xblock_services/video_config.py

```diff
@@ -0,0 +1,209 @@
+"""
+Video Configuration Service for XBlock runtime.
```

Comment:
Many of the other XBlock services are defined within the apps that related to them. For example, EnrollmentsService is in the enrollments app, CreditService is defined in the credit app, etc. Could you follow that pattern, and define this service within the video_configuration app? It might allow you to remove some of the method-level imports, which I assume you are using in order to get around importlinter and cyclical imports.